src/garage/tf/models/categorical_mlp_model.py

```diff
@@ -72,4 +72,5 @@ def _build(self, state_input, name=None):
 
         """
         prob = super()._build(state_input, name=name)
-        return tfp.distributions.OneHotCategorical(prob)
+        prob = prob / tf.math.reduce_sum(prob)
```

Comment:
i think you should probably use https://www.tensorflow.org/api_docs/python/tf/norm `ord=1` rather than a division operator.

your implementation is dangerous if `prob` is very small or zero. i think tf.norm should handle this? but maybe not.

Reply:
How about we manually set the `output_nonlinearity` of the MLP to softmax?

Reply:
I realized we were already using softmax so the output of the MLP should already be normalized and sum to 1?

Reply:
I can add a test

Reply:
softmax should sum to 1